src/main/resources/assets/vulkanmod/lang/ru_ru.json

```
@@ -0,0 +1,40 @@
+{
+  "vulkanmod.options.unknown": "Неизвестно",
+
+  "vulkanmod.options.pages.video": "Видео",
+  "vulkanmod.options.pages.graphics": "Графика",
+  "vulkanmod.options.pages.optimizations": "Оптимизация",
+  "vulkanmod.options.pages.other": "Другое",
```

Обычно переводится как `прочие настройки`, `дополнительные настройки`, `прочее`

```suggestion
  "vulkanmod.options.pages.other": "Прочее",
```